### ISSUE TITLE
Crashes upon inject

### ISSUE THREAD

**Issue by Fr0go1** [bruhmoment21/cs2-sdk#130]

**Comment by bruhmoment21** [comment]
are you using v2? and how are you injecting?

**Comment by Fr0go1** [comment]
> are you using v2? and how are you injecting?

Yes I am using v2 and I'm using manual map to inject. I tryed using multiple injectors that all used manual map.

**Comment by bruhmoment21** [comment]
Have you built with `/Zc:threadSafeInit-` flag as stated in README?
